lib/bloc_record/utility.rb

```diff
@@ -9,6 +9,7 @@ def camel_case(snake_cased_word)
 		end
 
 		def convert_keys(options)
+			puts options
```

Comment:
Remember to remove this when you are done debugging. I like to add `#TODO - Remove this` comments where there are things I need to remember to remove, whether they are print statements or other debugging logic.